lib/ObjWriter/.nuget/toolchain.osx.10.10-x64.Microsoft.DotNet.ObjectWriter.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>toolchain.osx.10.10-x64.Microsoft.DotNet.ObjectWriter</id>
-    <version>1.0.17-prerelease-00002</version>
+    <version>1.0.17-prerelease-00003</version>
```

Comment:
We added an export altering the public surface area. We should bump to `1.0.18-rerelease-00001`